examples/all-clusters-app/linux/include/bridged-actions-stub.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/****************************************************************************
+ * @file
+ * @brief Stub implementation for the Bridged Actions Cluster
+ ***************************************************************************/
```

Comment:
nit: consider if these `@file` comments are useful: the file is named "bridged-actions-strub" so it seems reasonably obvious that they include a stub implementation for bridged actions. The only extra info this gives is tthat "bridged actions" is a custer.

Reply:
I removed the @file section